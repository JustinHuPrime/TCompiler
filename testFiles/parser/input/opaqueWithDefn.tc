module foo;

struct bar {
  int i;
};