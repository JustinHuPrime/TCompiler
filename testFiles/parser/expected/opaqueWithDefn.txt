testFiles/parser/opaqueWithDefn.td (declaration):
FILE(1, 1, STAB(ENTRY(bar, OPAQUE(testFiles/parser/opaqueWithDefn.td, 3, 1, REFERENCES(testFiles/parser/opaqueWithDefn.tc, 3, 1)))), MODULE(1, 1, ID(1, 8, foo, REFERENCES())), OPAQUEDECL(3, 1, ID(3, 8, bar, REFERENCES(testFiles/parser/opaqueWithDefn.td, 3, 1))))
